liblll/CodeFragment.cpp

```diff
@@ -113,6 +113,22 @@ CodeFragment::CodeFragment(sp::utree const& _t, CompilerState& _s, bool _allowAS
 	}
 }
 
+namespace
+{
+/// Returns true iff the instruction is valid as a function. Assumes @a us is not nullptr.
```

Comment:
Sorry, but `Assumes @a us is not nullptr.` is wrong. The string cannot be nullptr, but can be empty. Anyway, there are not assumptions about the argument, because `.find()` will work for any input.

Reply:
Updated :)